site/docs/master/on-premises.md

```diff
@@ -17,8 +17,55 @@ For example, if you use [Portworx][4] for persistent storage, you can install th
 
 If there is no native snapshot plugin available for your storage platform, you can use Velero's [restic integration][1], which provides a platform-agnostic file-level backup solution for volume data.
 
+### Air-gapped deployments
+
+In an air-gapped deployment, there is no access to the public internet, and therefore no access to public container registries.
+
+In these scenarios, you will need to make sure that you have an internal registry, such as [Harbor][5], installed and the Velero core and plugin images loaded into your internal registry.
+
+Below you will find instructions to downloading the Velero images to your local machine, tagging them, then uploading them to your custom registry.
+
+#### Preparing the Velero image
```

Comment:
also need to do this process for the restic restore helper image if they're using it - `velero/velero-restic-restore-helper` (uses $VELERO_VERSION for tagging)